### PR TITLE
Docs for user bookmark count api added

### PR DESCRIPTION
**Pull request by sambuddhabasu**

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -552,6 +552,53 @@ Example
...
+/:username/stats/bmarkcount
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Usage
+''''''
+*GET* `/api/v1/admin/stats/bmarkcount`
+
+Get a json dump of the bookmark count for a user's account for a time period. The 
+time period can be specified or else a json dump of the bookmark count of the past 30 days will be returned.
+
```
Comment (mitechie) [inline docs/api.rst:564]: You can note the month trick as well. 

If a start date, with no end date, is provided and that date is the first day of a month, then the counts for that month will be provided.

**Comment by mitechie** [comment]
Cool, this looks good with the one note provided. 

Can you pull this into the existing branch. Then we'll QA, test, and land as one whole the api work including the views, the docs, and the tests.

**Comment by sambuddhabasu** [comment]
Sure after the necessary changes are done, it will be pulled to the existing graphapi branch.
Also, this issue can be closed then.

**Comment by mitechie** [comment]
superceded.

**Pull request closed without merging**